ReactiveCocoaTests/Swift/SignalSpec.swift

```diff
@@ -496,6 +495,71 @@ class SignalSpec: QuickSpec {
 			}
 		}
 
+		describe("Signal.merge") {
+			it("should emit values from all signals") {
+				let (signal1, observer1) = Signal<Int, NoError>.pipe()
+				let (signal2, observer2) = Signal<Int, NoError>.pipe()
+
+				let mergedSignals = Signal.merge([signal1, signal2])
+
+				var lastValue: Int?
+				mergedSignals.observeNext { lastValue = $0 }
+
+				expect(lastValue).to(beNil())
+
+				observer1.sendNext(1)
+				expect(lastValue) == 1
+
+				observer2.sendNext(2)
+				expect(lastValue) == 2
+
+				observer1.sendNext(3)
+				expect(lastValue) == 3
+			}
+
+			it("should not stop when one signal completes") {
```

Comment:
Sure, but I just wanted high level tests. That should be covered by `flatten(.Merge)`, so I  didn't want to duplicate too much of that coverage here.